webapp/graphite/render/grammar.py

```diff
@@ -1,151 +1,17 @@
-from pyparsing import (
-    Forward, Combine, Optional, Word, Literal, CaselessKeyword,
-    CaselessLiteral, Group, FollowedBy, LineEnd, OneOrMore, ZeroOrMore,
-    alphas, alphanums, printables, delimitedList, quotedString, Regex,
-    __version__, Suppress, Empty
-)
+# Copyright 2020 Liberty Global B.V.
```

Comment:
I work for Liberty Global and I investigated the issue, found the problem and developed the fix at work.

The project is licensed under Apache 2.0 so there shouldn't be a problem with such contribution, at least I've never had any problems contributing to other projects using this license.

So what's the problem with this copyright notice here?

Reply:
Does @pengyusong work for LGI as well?

Reply:
Honestly I don't know, we don't know each other, but I doubt that he works for LGi.

Reply:
IANAL, and I really do not want to have this discussion. If she/he created the fix, you copied and modified it then I think your copyright claim is questionable at best. 

To get to the point copyright != license, having to deal with 100's of copyrightholders can be a pita for open source projects, especially if they wan't to change the license later on. Please just remove that line so that everybody can go back to doing cool stuff. Liberty Global does not have a policy that you have to include that line (well maybe that changed recently, but wasn't the case when I was working for them).

Reply:
IANAL either and I get your point. The problem is that we do have a written policy for putting copyright notices in code. I'm not sure if it's global but it applies to my project. Nevertheless I got the green light to skip it this time, so I just dropped the copyright notice from the commit.

Reply:
Thank you!!